modules/model/item/components/overcast.js

```diff
@@ -27,7 +27,14 @@ export class OvercastItemModel extends BaseItemModel {
               additional  : new fields.StringField(),
               characteristic  : new fields.StringField(),
               bonus : new fields.BooleanField(),
-         }),
+          }),
+          usage: new fields.SchemaField({
```

Comment:
I'm not 100% sure yet this is needed. I just know that I had some issues about non-existing `overcast.usage` when I tried casting a spell that's not physically on Actor.

Will test more and if works without it, I will probably rever this